templates/keystoneapi/config/keystone.conf

```diff
@@ -5,9 +5,13 @@ use_stderr=true
 {{if .memcachedTLS}}
 backend = dogpile.cache.pymemcache
 memcache_servers={{ .memcachedServers }}
+enable_retry_client = true
+retry_attempts = 2
+retry_delay = 0
```

Comment:
I think there should be at least _some_ delay between the retries, however small.

Reply:
this is just the default value as set in https://docs.openstack.org/keystone/latest/configuration/samples/keystone-conf.html , plan is to surface this in the docs so that users can set it to the desired value.